example/example.swagger.yaml

```diff
@@ -190,6 +190,80 @@ paths:
             type: string
             pattern: /\d+/
             description: GET /v1/stream parameter
+  /v1/upload:
+    post:
+      responses:
+        "200":
+          description: POST /v1/upload Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    enum:
+                      - success
+                  data:
+                    type: object
+                    properties:
+                      files:
+                        type: object
+                        additionalProperties:
+                          type: object
+                          properties:
+                            name:
+                              type: string
+                            size:
+                              type: integer
+                              format: int64
+                              minimum: 0
+                              exclusiveMinimum: false
+                              maximum: 9007199254740991
+                              exclusiveMaximum: false
+                            mime:
+                              type: string
+                            hash:
+                              type: string
+                          required:
+                            - name
+                            - size
+                            - mime
+                            - hash
+                    required:
+                      - files
+                required:
+                  - status
+                  - data
+        "400":
+          description: POST /v1/upload Error response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    enum:
+                      - error
+                  error:
+                    type: object
+                    properties:
+                      message:
+                        type: string
+                    required:
+                      - message
+                required:
+                  - status
+                  - error
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties: {}
+              required: []
```

Comment:
It would be great to have file inputs here...
So it has to be the endpoint input instead of options provided by middleware

Reply:
fixed in 065f8e9